### DISCUSSION TITLE
Exciting Opportunity: Angular Developer Internship at Bayslope Business Solutions

### DISCUSSION THREAD

**Discussion by milind2thakur** [community/community#144399] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Role Description
This is a full-time, hybrid internship opportunity for an **Angular Developer Intern** based in Gurugram. As an Angular Developer Intern, you will have the chance to enhance your Angular skills by working on real-world projects. This hybrid role is primarily remote, with the flexibility to work from the office two days a week.

Compensation: **Rs. 15,000 per month**
Location: 2 days per week at WeWork Two Horizon Center, Gurugram
Duration: 2 months (extendable up to an additional 4 months)
Qualifications
Front-End Development Skills
Experience deploying at least 3 personal or academic projects with Angular-based front ends
Strong understanding of Angular Material, TypeScript, HTML, and CSS
Excellent problem-solving and debugging skills
Strong communication skills
Ability to work independently
Bachelor's degree in Computer Science, Engineering, or a related field
Additional qualifications that would be beneficial:

Prior experience in similar internships within the last 6 months
Familiarity with Google Cloud Platform (Firestore, Authentication, Functions, and Storage)

If you're interested, please contact me at **milind@bayslope.com**.

**Comment by samus-aran** [comment]
Hi @milind2thakur , we’re glad you’re here 😀

Thanks for sharing your job opportunity with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
